servicemesh-install/topics/install-multi-tenant.adoc

```diff
@@ -0,0 +1,73 @@
+* The Jaeger agent runs as a `DaemonSet`, therefore `tracing` may only be enabled for a single `ServiceMeshControlPlane` instance.
+* Automatic route creation is currently incompatible with multi-tenant {ProductShortName} installations. Ensure that it is disabled in your `ServiceMeshControlPlane` if you plan to attempt a multi-tenant installation.
+* If you Delete the project that contains the control plane before you delete the `ServiceMeshControlPlane` resource, some patrs of the installation may not be removed:
+** Service accounts added to the `SecurityContextConstraints` may not be removed
+** `OAuthClient` resources associated with Kiali may not be removed, or its list of redirectURIs may not be accurate
+
+== Comparison of multi-tenant and cluster-wide installations
+
+The main difference between a multi-tenant installation and a cluster-wide installation is the scope of privileges used by the control plane deployments, for example, galley and pilot. The components no longer use cluster-scoped Role Based Access Control (RBAC) `ClusterRoleBinding`, but rely on namespace-scoped RBAC `RoleBinding`. Every namespace in the `members` list will have a `RoleBinding` for each service account associated with a control plane deployment and each control plane deployment will only watch those member namespaces. Each member namespace has a `maistra.io/member-of` label added to it, where the `member-of` value is the namespace containing the control plane installation.
+
+== Configuring a multi-tenant installation
+
+You can configure a multi-tenant installation by setting the `multitenant` option to `true` in the istio section of the `ServiceMeshControlPlane` resource. For example,
+
+.Multi-tenant custom resource example
+
+[source,yaml]
+----
+  apiVersion: maistra.io/v1
+  kind: ServiceMeshControlPlane
+  spec:
+    istio:
+    # enable multitenant
+    	multitenant: true
+       #additional control plane configuration details
+----
+
+== Configuring {ProductName} namespaces in multi-tenant installations
+
+A multi-tenant control plane installation only affects namespaces configured as part of the {ProductShortName}. You must specify the namespaces associated with the {ProductShortName} in a `ServiceMeshMemberRoll` resource located in the same namespace as the `ServiceMeshControlPlane` resource and name it `default`. Here is an example that joins the bookinfo namespace into the {ProductShortName}:
+
+.Namespace configuration example
+
+[source,yaml]
+----
+  apiVersion: maistra.io/v1
+  kind: ServiceMeshMemberRoll
+  metadata:
+    name: default
+  spec:
+    members:
+    # a list of namespaces joined into the service mesh
+    - bookinfo
+----
+
+You can add any number of namespaces, but a namespace can only belong *one* `ServiceMeshMemberRoll`.
```

Comment:
belong to *one*

Reply:
I believe bold is a better choice than italics for this.

Reply:
Not sure where the italics came from. Sentence is currently can only belong *one*, there should be a to in there.